docs/source/common/weights_loading.rst

```diff
@@ -6,30 +6,32 @@
 
 .. _weights_loading:
 
-##########################
-Saving and loading weights
-##########################
+##############################
+Saving and Loading Checkpoints
+##############################
 
-Lightning automates saving and loading checkpoints. Checkpoints capture the exact value of all parameters used by a model.
+Lightning provides functio to save and load checkpoints.
 
 Checkpointing your training allows you to resume a training process in case it was interrupted, fine-tune a model or use a pre-trained model for inference without having to retrain the model.
 
 
 *****************
-Checkpoint saving
+Checkpoint Saving
 *****************
+
 A Lightning checkpoint has everything needed to restore a training session including:
 
-- 16-bit scaling factor (apex)
+- 16-bit scaling factor (if using APEX mixed-precision)
```

Comment:
Is the scaling factor not saved when using native amp?

Reply:
yeah for both. .updated!!